package.json

```diff
@@ -80,6 +80,9 @@
     "build-compile": "babel src -d dist --source-maps inline --copy-files",
     "build-script": "browserify src/bundle.js --debug -o dist/lumagl.js -t babelify -t glslify && cat dist/lumagl.js | uglifyjs > dist/lumagl.min.js",
     "docs": "npm run build-docs && open ./_docs/index.html",
+    "gh-pages": "git subtree push --prefix _docs origin gh-pages",
```

Comment:
Maybe have a common 'doc' prefix for all these scripts. When the list of script grows it gets easy to lose track of what's what.
```
doc-gh-pages
doc-gh-pages-force
doc-publish
 ```